vector/src/main/java/im/vector/util/BitmapUtil.kt

```diff
@@ -78,7 +80,14 @@ fun Bitmap.createSquareBitmap(): Bitmap? = when {
  */
 fun Bitmap.addBackgroundColor(backgroundColor: Int): Bitmap {
     // Create new bitmap based on the size and config of the old
-    val newBitmap = Bitmap.createBitmap(width, height, config)
+    val newBitmap = Bitmap.createBitmap(width, height, config ?: Bitmap.Config.ARGB_8888)
+
+    // Reported by the Play Store
```

Comment:
it's in the commit message and in the CHANGES.rst. I've added a comment to avoid another developer (or future me) to remove this code which look useless :)